### PR TITLE
Makes mice chew wires less often, alternative to time-green

### PR DESCRIPTION
**Pull request by ghost**
makes mice chew wires less often by way of yet more prob checks because while more things to do for engis are good usually they're too occupied by something exploding to fix a small wire break and you yourself can't fi it without insu's because trying to lay down the wire removes the entire wire if you get shocked
#### Changelog

:cl:
rscadd: Mice have a slight self preservation instinct and will no longer eat wires incessantly.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 1]
```diff
@@ -60,19 +60,19 @@
...
-			if(C && prob(15))
-				if(C.avail())
-					visible_message("<span class='warning'>[src] chews through the [C]. It's toast!</span>")
-					playsound(src, 'sound/effects/sparks2.ogg', 100, 1)
-					C.Deconstruct()
-					death(toast=1)
-				else
-					C.Deconstruct()
-					visible_message("<span class='warning'>[src] chews through the [C].</span>")
+				if(C && prob(5))
```
Comment (Time-Green) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm:67]: Why the tab here?

**Comment by Time-Green** [comment]
The original chance was 0.15%
You changed it to 0.25%

This actually increases the chance.

**Comment by ghost** [comment]
there's a prob check and then a prob check, time. that's the tab there.

it has a 0.25% chance and then a 0.25% chance within that 0.25 chance.

**Comment by Time-Green** [comment]
There's 2 prob checks.
The original was 1% and then 15%
0.01 x 0.15 = 0.0015 Wich converts back to 0.15%
The new one does 5% and then 5% again.
0.05 x 0.05 = 0.0025 = 0.25%

**Comment by Super3222** [comment]
so instead of just asking for the changes and working with the first pr, you decided to make an entirely new one, why?

i feel like you were extremely pissed off while making these changes

**Pull request closed without merging**